packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -295,6 +296,14 @@ export interface InstanceProps {
    * @default - public IP address is automatically assigned based on default behavior
    */
   readonly associatePublicIpAddress?: boolean;
+
+  /**
+   * Specifying the CPU credit type for burstable EC2 instance types (T2, T3, T3a, etc).
+   * T3 instances with `host` tenancy do not support the unlimited CPU credit option.
```

Comment:
This document appears to be taken from the L1 or CFn text.
The `host` here refers to the `tenancy` property of `CfnInstanceProps`, which is not visible in the L2 construct. Wouldn't it therefore be more appropriate to use the word `dedicated host`?

Otherwise, users may be get confused (like me).

Reply:
@go-to-k 
Indeed, you're right. I've updated the sentence, so please check it.